safe_cli/operators/safe_tx_service_operator.py

```diff
@@ -32,6 +36,58 @@ def __init__(self, address: str, node_url: str):
     def approve_hash(self, hash_to_approve: HexBytes, sender: str) -> bool:
         raise NotImplementedError("Not supported when using tx service")
 
+    def sign_message(
+        self,
+        eip191_message: Optional[str] = None,
+        eip712_message_path: Optional[str] = None,
+    ) -> bool:
+        if eip712_message_path:
+            try:
+                message = json.load(open(eip712_message_path, "r"))
+                message_hash = eip712_encode_hash(message)
+            except ValueError:
+                raise ValueError
+        else:
+            message = eip191_message
+            message_hash = defunct_hash_message(text=message)
+
+        safe_message_hash = self.safe.get_message_hash(message_hash)
+        eoa_signers, hw_wallet_signers = self.get_signers()
+        signers = []
+        signatures = b""
+        for eoa_signer in eoa_signers:
+            signature_dict = eoa_signer.signHash(safe_message_hash)
+            signature = signature_to_bytes(
+                signature_dict["v"], signature_dict["r"], signature_dict["s"]
+            )
+            signers.append(eoa_signer.address)
+            signer_pos = sorted(signers, key=lambda x: int(x, 16)).index(
```

Comment:
You can build `SafeSignature` objects and then use the new method `export_signatures` on `safe-eth-py`